app/oauth.js

```diff
@@ -0,0 +1,102 @@
+'use strict'
```

Comment:
This whole file seems to be a duplicate of the oauth model from the server — delete!